inference/engine.py

```diff
@@ -392,6 +392,7 @@ def preprocess(self, sents: List[str], lang: str):
     def postprocess_batch(self, translations: List[str], lang: str, input_sents: List[str] = None, placeholder_entity_map: List[dict] = None) -> List[str]:
         """
         Wrapper function over `postprocess` that postprocesses a batch of translations.
+        FIXME: Do we really need this method?
```

Comment:
Can you please remove this function and directly use the underlying function call directly wherever this function is being called?